woocommerce/woocommerce-android
# Add toolbar to ShippingLabelRefundFragment

Part of #10570 

### Description

This PR introduces individual toolbars for ShippingLabelRefundFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> click on 3 dots menu next to package name -> Request a refund) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="244" alt="Screenshot 2024-02-12 at 6 37 36 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/02daabb9-1ff6-4e87-a585-b04ce534f21c">
